app/controllers/items_controller.rb

```diff
@@ -1,37 +1,37 @@
 class ItemsController < ApplicationController
-    before_action :authenticate_user!
-    before_action :set_group
+  before_action :authenticate_user!
+  before_action :set_group
 
-    def index
-        @items = Group.find(@group.id).items.order(created_at: :desc)
-    end
+  def index
+    @items = Group.find(@group.id).items.order(created_at: :desc)
+  end
```

Comment:
- [ ] Kindly design the `Items#Index` page to look identical to that of the Behance Design provided. Apart from the layout, kindly consider the following when designing this page:

![image](https://github.com/yegonkimutai/budget-app/assets/77106962/e81e24ab-e94a-43d9-bbc9-9f881f3f2574)

#### N.B: Please note that the Add Transaction button should be situated at the bottom of the Page as highlighted in the screenshot above. 👍 ⭐ 

| Your Design  | Behance Design |
| ------------- | ------------- |
| ![image](https://github.com/yegonkimutai/budget-app/assets/77106962/42f19097-bb3e-4d2b-aa36-69f49824a862) | ![image](https://github.com/yegonkimutai/budget-app/assets/77106962/47823852-cde4-41a9-b072-1b84ef0109f8) |